pkg/controller/rds/dbcluster/setup.go

```diff
@@ -131,3 +152,42 @@ func filterList(cr *svcapitypes.DBCluster, obj *svcsdk.DescribeDBClustersOutput)
 	}
 	return resp
 }
+
+func (e *custom) savePasswordSecret(ctx context.Context, cr *svcapitypes.DBCluster, pw string) error {
+	if cr.Spec.ForProvider.MasterUserPasswordSecretRef == nil {
+		return errors.New("no MasterUserPasswordSecretRef given, unable to save password")
+	}
+	ref := cr.Spec.ForProvider.MasterUserPasswordSecretRef
+	nn := types.NamespacedName{
+		Name:      ref.Name,
+		Namespace: ref.Namespace,
+	}
+	sc := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      nn.Name,
+			Namespace: nn.Namespace,
+		},
+	}
```

Comment:
```suggestion
	sc := &corev1.Secret{}
```

`Get` fills that for you.